mep/people/tests/test_models.py

```diff
@@ -128,6 +128,17 @@ def test_nationality_list(self):
         pers.nationalities.add(country)
         assert pers.list_nationalities() == 'France, Spain'
 
+    def test_account_id(self):
+        # create a person
+        pers = Person.objects.create(name='Foobar')
+        # create an account
+        acct = Account.objects.create()
+        # not associated so person has no account number
+        assert pers.account_id() == ''
+        # associate
+        acct.persons.add(pers)
+        pers.account_id() == acct.id
```

Comment:
this is just missing the `assert` keyword, right?

Reply:
Sigh, 🤦‍♂ . Yep, will get that.